src/game/traps/movable_block.h

```diff
@@ -2,25 +2,17 @@
 
 #include "global/types.h"
 
-#include <stdint.h>
-
-typedef enum {
-    MBS_STILL = 1,
-    MBS_PUSH = 2,
-    MBS_PULL = 3,
-} MOVABLE_BLOCK_STATE;
-
-extern int16_t g_MovingBlockBounds[12];
-
-void SetupMovableBlock(OBJECT_INFO *obj);
-void InitialiseMovableBlock(int16_t item_num);
-void MovableBlockControl(int16_t item_num);
-void MovableBlockCollision(
+void MovableBlock_Setup(OBJECT_INFO *obj);
+void MovableBlock_Initialise(int16_t item_num);
+void MovableBlock_Control(int16_t item_num);
+void MovableBlock_Collision(
     int16_t item_num, ITEM_INFO *lara_item, COLL_INFO *coll);
-void DrawMovableBlock(ITEM_INFO *item);
+void MovableBlock_Draw(ITEM_INFO *item);
 
-int32_t TestBlockMovable(ITEM_INFO *item, int32_t blockhite);
-int32_t TestBlockPush(ITEM_INFO *item, int32_t blokhite, uint16_t quadrant);
-int32_t TestBlockPull(ITEM_INFO *item, int32_t blokhite, uint16_t quadrant);
+bool MovableBlock_Test(ITEM_INFO *item, int32_t blockhite);
```

Comment:
It is easy to tell what `MovableBlock_TestPush` and `MovableBlock_TestPull` are checking. But `MovableBlock_Test` is not clear IMO.

Reply:
I'm not sure what it's testing TBH.

Reply:
Ok so ChocolateFan says it is checking if a block can be moved by testing the square that the block is going to be in. If it has the same height as the square the block is in now. Possible suggestions:
`MovableBlock_TestDestination`
`MovableBlock_TestHeight`
` MovableBlock_TestMovement`

Reply:
Applied